pitmutationmate/build.gradle.kts

```diff
@@ -65,4 +65,11 @@ tasks {
     publishPlugin {
         token.set(System.getenv("PUBLISH_TOKEN"))
     }
+
+    register<org.jetbrains.intellij.tasks.RunIdeTask>("androidRunIde") {
```

Comment:
I think `runIdeAndroid` is more fitting and in line with other task names...
Or even `runIdeAndroidStudio`